R/parse-body.R

```diff
@@ -128,7 +128,8 @@ parser_json <- function() {
     charset <- getCharacterSet(content_type)
     value <- rawToChar(value)
     Encoding(value) <- charset
-    safeFromJSON(value)
+    args <- safeFromJSON(value)
+    combine_keys(args, FALSE)
```

Comment:
I'm leaning towards saying we shouldn't do this here. There's nothing wrong with returning multiple keys with the same name. I would image the route should handle it.

```r
jsonlite::parse_json('{"a":1,"a":2}')
# $a
# [1] 1
# 
# $a
# [1] 2
```

Reply:
It won't because only the first one will be matched there
https://github.com/rstudio/plumber/blob/20427b16eefed4a16952d22346ca41021d09cfce/R/plumber-step.R#L125

Also this gives an error:
```r
> a <- function(b, b) {}
Error: repeated formal argument 'b' on line 1
```

Reply:
Are you saying we should do it somewhere else where we match args with the route?